pages/index.js

```diff
@@ -175,22 +175,28 @@ export default class Index extends React.Component<Props, State> {
       Object.keys(translations)
         .filter(key => /otherMetaTags/.test(key))
         .reduce((result, key) => ({ ...result, [key]: translations[key] }), {
-          metaDescription: translations.metaDescription,
-          metaTitle: translations.metaTitle,
+          metaDescription: translations?.metaDescription,
+          metaTitle: translations?.metaTitle,
         });
 
     // TODO: introduce optional chaining to the app for undefined check
```

Comment:
This is done now, right? Can we remove this comment?

Reply:
yes, will fix it